flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/tiered/storage/TieredStorageMemoryManagerImpl.java

```diff
@@ -244,18 +246,8 @@ public int getMaxNonReclaimableBuffers(Object owner) {
     public boolean ensureCapacity(int numAdditionalBuffers) {
         checkIsInitialized();
 
-        final int numRequestedByGuaranteedReclaimableOwners =
```

Comment:
Is this a bug fix?  We'd better separate it as a hotfix commit.

Reply:
It's just a code refactor in the past, currently, I think this change is meaningless, so I reverted it.